docs/code.rst

```diff
@@ -203,7 +204,6 @@ Sensors
 .. autoclass:: airflow.contrib.sensors.pubsub_sensor.PubSubPullSensor
 .. autoclass:: airflow.contrib.sensors.qubole_sensor.QuboleSensor
 .. autoclass:: airflow.contrib.sensors.redis_key_sensor.RedisKeySensor
-.. autoclass:: airflow.contrib.sensors.sftp_sensor.SFTPSensor
```

Comment:
Why did you remove this one?

Reply:
I didn't now that was removed. Will ensure it's fixed